src/deployment/deploy.py

```diff
@@ -305,7 +304,7 @@ def get_instance_url(self) -> str:
         # The url to access the instance
         # This also represents the legacy identifier_uris of the application
         # registration
-        if self.multi_tenant_domain:
+        if self.multi_tenant_domain != "":
```

Comment:
Previous code should be fine, or you hit an issue with it ? 
https://docs.python.org/3/library/stdtypes.html#truth-value-testing

Reply:
No I want this param to be required in the config.json. This way, if an instance isn't multi-tenant, they can just leave it blank. So in this case self.multi_tenant_domain is always set to at least "". The previous check is therefore not sufficient.